unity-client/Assets/Builder/Scripts/Gizmos/DCLBuilderGizmo.cs

```diff
@@ -27,16 +27,24 @@ public abstract class DCLBuilderGizmo : MonoBehaviour
         public abstract void SetSnapFactor(DCLBuilderGizmoManager.SnapInfo snapInfo);
         public abstract void TransformEntity(Transform targetTransform, DCLBuilderGizmoAxis axis, float axisValue);
 
-        public virtual void Initialize(Camera camera)
+
+        public virtual void Initialize(Camera camera,Transform _cameraTransform)
```

Comment:
```suggestion
        public virtual void Initialize(Camera camera, Transform _cameraTransform)
```

Also `_cameraTransform` doesn't follow our name convention.